### ISSUE TITLE
Feedbacks from community (testing)

### ISSUE THREAD

**Issue by dyoann** [AigangNetwork/aigang-platform-web#55]
You'll find all the infos in this document, I prepared it to help you ;)
I'll add any  of the upcoming feedbacks in attachement when I got some more.

The link of the current document is  : 
https://docs.google.com/document/d/1gM1j_Uj5LdIP7G7ccqC7F-xwHLxDoLIZMDbDR7G73jU/edit?usp=sharing

Here is the first feedback I got from someone 👍 : 
[Aigang platform feedback.docx](https://github.com/AigangNetwork/aigang-platform-web/files/2048901/Aigang.platform.feedback.docx)

**Comment by dariusdev** [comment]
Thank you dyoann :) very useful job 👍

**Comment by dyoann** [comment]
[list of the tasks to be done.docx](https://github.com/AigangNetwork/aigang-platform-web/files/2066835/list.of.the.tasks.to.be.done.docx)

Another one but it seems he didn’t understand how to upload a dataset / model :(
Maybe the documentation link should be somewhere in the webiste.

**Comment by tchun21** [comment]
[Gr8Wall%2FTC Feedback.docx](https://github.com/AigangNetwork/aigang-platform-web/files/2077298/Gr8Wall.2FTC.Feedback.docx)

**Comment by dyoann** [comment]
Great feedback :)

**Comment by dyoann** [comment]
@dariusdev If it was promoted by the official team, it would have been more useful, right ?
 I thought this thread would be more alive, but it's dead and the last review which had interesting reviews was ignored , it's disappointing really.

I think also some people didn’t want to participate at this stage for these reasons : 
- There was no money/reward offered to complete this job
- The features to be tested seemed to easy for them and they didn't want to commit (at least for free)
- The price tanked too much, so they don't believe anymore in the project
- They don't care about anything else than the price of the token
- I did the work myself and I’m not part of Aigang so I'm not taken seriously

In any case, I still think this document was a good starting base to improve the system, and also in the future when it has more functionalities.

**Comment by dariusdev** [comment]
Currently, we are working to provide more features for users: predictions market and first production-ready android battery product. After we will have a working concept in QA we return to all these products integrations, testing and these feedbacks changes.

**Comment by dyoann** [comment]
Ok, good to hear.
So if I understand well, it's supposed to be useful in some future , right ? (Or did I waste my time ? )
Thanks

**Comment by dariusdev** [comment]
Right. When we will finish coding this features we return to this issue. It is not waste of your time :) Issue is not closed too.
